execution-plane/arcon/src/util/prost_helpers.rs

```diff
@@ -0,0 +1,21 @@
+use prost::Message;
+
+// it's so stupid that this is needed
```

Comment:
This comment doesn't really add anything ^^
A short comment on why it is needed would be better.

Reply:
Whoops, sorry. I was using `Option<T>` as a state in some test, and prost doesn't really do well with most std types.

Reply:
Ah, nevermind. It wasn't even a test. It was needed to port `IncrementalWindow` to use state backends.